src/sentry/static/sentry/app/components/assigneeSelector.jsx

```diff
@@ -79,7 +79,7 @@ const AssigneeSelector = createReactClass({
     if (nextProps.id != this.props.id || loading != this.state.loading) {
       let group = GroupStore.get(this.props.id);
       this.setState({
-        assignedTo: group.assignedTo,
+        assignedTo: group && group.assignedTo,
```

Comment:
Do we know why `group` can be undefined here?

Reply:
I think it's a race condition on load? to be honest I don't so maybe this fix is too hasty